.github/workflows/specs.yml

```diff
@@ -3,24 +3,22 @@ on: [push, pull_request]
 jobs:
   tests:
     strategy:
+      fail-fast: false
       matrix:
         os: [ubuntu-latest, macos-latest]
-        ruby: [2.7]
     runs-on: ${{ matrix.os }}
     steps:
       - uses: ruby/setup-ruby@v1
         with:
-          ruby-version: ${{ matrix.ruby }}
+          ruby-version: 2.7
       - uses: actions/checkout@v2
-      - name: Specs
-        run: |
-          gem install tf -v '>=0.4.1'
-          hash -r
-          ./install
-          git submodule init
-          git submodule update
-          source ~/.rvm/scripts/rvm
-          rvm reload
```

Comment:
Are you sure `rvm reload` is not needed?

Reply:
GH Actions steps are all isolated, they don't run in the same bash process.